### ISSUE TITLE
Wrong Paths

### ISSUE THREAD

**Issue by rainerhahnekamp** [softarc-consulting/sheriff#16]
When a @path settings in tsconfig or path in sheriff.config.ts does not exist, we should get a better error message.

At the moment, the message mentions `FsPath` which is not very explanatory.

The same could happen, if an import points to a file that does not exist.

Error message should be something like:

```bash
`project/src/main.ts` imports `./customers/customers.ts` which does not exist.
```

**Comment by RobinMeow** [comment]
I think, this is relating, but not exactly the same. 

https://github.com/softarc-consulting/sheriff/issues/49

**Comment by rainerhahnekamp** [comment]
Fixed in #ff3b3a846807440cdb4fe8a1905e098e4a5e5ba5
